.angulardoc.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Can you please provide some additional info about this?

Reply:
Hi, sorry, pushed this file by mistake. this is for VSCode plugin for angular projects (https://marketplace.visualstudio.com/items?itemName=AngularDoc.angulardoc-vscode)
It shows all angular modules and their heirarchy